### DISCUSSION TITLE
Horizon Radio Offline?

### DISCUSSION THREAD

**Discussion by Metalraptor** [tomassasovsky/radio-horizon.dart#114] [Q&A]
For about a month now the Horizon app has been listed as offline, and nothing seems to work. I've tried removing the app and then re-inviting it to the sever, but it remains in the offline state. Is the app still working, or is this an issue on my end?

**Comment by tomassasovsky** [comment]
Hey there!

I'm sorry for this, the bot crashed a while ago and I postponed fixing it because of my current workload. I also needed to reconfigure my server.  I just turned it back on, so you should be able to use it now!

**Comment by Metalraptor** [comment]
No worries, just wanting to make sure if it was still working. 
Thanks for fix fixing it. ^_^
